src/platform/src/common/features/user-settings/tabs/Profile.jsx

```
@@ -55,355 +56,258 @@ const Profile = () => {
     const userID = session.user.id;
-
-    // Create a cleaned version of userData without empty profilePicture
-    const dataToUpdate = { ...userData };
-    if (!dataToUpdate.profilePicture) {
-      delete dataToUpdate.profilePicture;
-    }
+    const payload = {
+      ...userData,
+      profilePicture: localImagePreview || userData.profilePicture,
+    };
+    if (!payload.profilePicture) delete payload.profilePicture;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)